File: seed/java-sdk/path-parameters/src/main/java/com/snippets/Example14.java
package com.snippets;

import com.seed.api.SeedApiClient;
import com.seed.api.resources.user.requests.UserGetUserMetadataRequest;

public class Example14 {
    public static void main(String[] args) {
        SeedApiClient client =
                SeedApiClient.builder().url("https://api.fern.com").build();

        client.user()
                .getusermetadata(
                        "tenant_id",
                        "user_id",
                        1,
                        UserGetUserMetadataRequest.builder().build());
    }
}

File: seed/swift-sdk/path-parameters/Snippets/Example14.swift
import Foundation
import Api

private func main() async throws {
    let client = ApiClient(baseURL: "https://api.fern.com")

    _ = try await client.user.getusermetadata(
        tenantId: "tenant_id",
        userId: "user_id",
        version: 1
    )
}

try await main()
